docling/models/tesseract_ocr_cli_model.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         self._version: Optional[str] = None
         self._tesseract_languages: Optional[List[str]] = None
         self._script_prefix: Optional[str] = None
+        self._is_auto: bool
```

Comment:
for safety, can you please assign a default value `False` (then the later assignment in `def _set_languages_and_prefix()`can overwrite it)